src/devices/machine/storyware.h

```diff
@@ -0,0 +1,74 @@
+// license:BSD-3-Clause
+// copyright-holders:QUFB
+/**********************************************************************
+
+    Storyware interface emulation, for use with the Sega Pico / Copera / Beena consoles.
+
+    Given a directory path to the booklet's pages (one PNG file for each page), the device will render them
+    on a dedicated screen, according to the current effective page. Uses the same pen input port
+    as the main / tablet screen so that movement is kept synchronized between screens.
+
+    To ensure accurate mappings with interactable regions defined in games, images should be cropped
+    to page borders and scaled to half the screen's width while preserving the screen ratio (e.g. 352/240).
```

Comment:
It will be scaled by `render_resample_argb_bitmap_hq` to fit that ratio in the allocated bitmaps. But if it makes sense, we can show a warning to the player if the ratio doesn't match, what would be the best way to do that?

Reply:
For something like that, I would probably go with `osd_printf_warning`. It's more important than verbose spew, but not quite to the level of an error. :)